dcmnet/include/dcmtk/dcmnet/scu.h

```diff
@@ -1066,6 +1064,9 @@ class DCMTK_DCMNET_EXPORT DcmSCU
     /// ACSE timeout (default: 30 seconds)
     Uint32 m_acseTimeout;
 
+    /// TCP Connection timeout (default: global dcmConnectionTimeout
+    Uint32 m_tcpConnectTimeout;
```

Comment:
@jogerh I guess this member variable should be of type Sint32, because a timeout of "unlimited" is specified by value of -1.

Reply:
Absolutely right. Fixed in [2349204](https://github.com/DCMTK/dcmtk/pull/61/commits/2349204a3b7327de0a19c39c274468cdb3035746)